draft-ietf-quic-transport.md

```diff
@@ -3204,9 +3204,15 @@ response to further packets that it receives.
 A receiver MUST discard a newly unprotected packet unless it is certain that it
 has not processed another packet with the same packet number from the same
 packet number space. Duplicate suppression MUST happen after removing packet
-protection for the reasons described in Section 9.3 of {{QUIC-TLS}}. An
-efficient algorithm for duplicate suppression can be found in Section 3.4.3 of
-{{?RFC4303}}.
+protection for the reasons described in Section 9.3 of {{QUIC-TLS}}.
+
+Endpoints that track individual packets for the purposes of detecting duplicates
+might accumulate excessive state.  The data required for detecting duplicates
```

Comment:
Would it be possible for us to recommend tracking ranges? For example, it might be a good idea to state like: _For the purposes of detecting duplicates, endpoints might track individual packets or ranges of consecutive packet numbers, though tracking individual packets might accumulate excessive state._

Reply:
I considered that, but I don't feel like a recommendation is necessary.  It's the natural implementation strategy because of how we structured ACK frames, so there is enough bias there.  But it crosses somewhat over from "here are the pitfalls" into "here is how you build a stack" too much for me.

Reply:
Thank you for elaborating. This works for me.